cmd/fission-bundle/main.go

```diff
@@ -78,14 +78,11 @@ func runMessageQueueMgr(logger *zap.Logger, routerUrl string) {
 	}
 }
 
-func runStorageSvc(logger *zap.Logger, port int, filePath string) {
-	subdir := os.Getenv("SUBDIR")
-	if len(subdir) == 0 {
-		subdir = "fission-functions"
+func runStorageSvc(logger *zap.Logger, port int, filePath string, storageArgs ...string) {
+	err := storagesvc.Start(logger, filePath, port, storageArgs...)
+	if err != nil {
+		logger.Fatal("error starting storage service", zap.Error(err))
 	}
-	enableArchivePruner := true
```

Comment:
Any specific reason to remove this from here - in future we could expose this via values so the user can define if pruning should be enabled etc.

Reply:
Same as above. If we want to get this values from `values.yaml` we will pass this on to `Start` function and handle it there. Like it is being handled now [here](https://github.com/fission/fission/pull/1577/files?file-filters%5B%5D=.go#diff-f33eed829a16e5e68e0b2c560b158512R53)